doc/api/thaliemitter.md

```diff
@@ -0,0 +1,189 @@
+# The `ThaliEmitter` class
+
+The `ThaliEmitter` class is a bridge between the native Cordova plugin and JavaScript to give you an easy to use interface for broadcasting availability and to connect and disconnect from peer clients.
+
+## Usage
+
+This is the basic usage to listen for the `peerAvailabilityChanged` event as well as start broadcasting the availability of the current device.
+
+```js
+var emitter = new ThaliEmitter();
```

Comment:
Wouldn't it be good to include the require statement so they know what name we use?